api/utils.go

```diff
@@ -846,3 +867,49 @@ func getFlumeTransactionReceiptsBlock(ctx context.Context, db *sql.DB, offset, l
 	return getFlumeTransactionReceiptsQuery(ctx, db, offset, limit, chainid, query, logsQuery, params...)
 
 }
+
+// wtdrlIndex MEDIUMINT,
+// vldtrIndex MEDIUMINT,
+// recipient VARCHAR(20),
+// amount    blob,
+// block     BIGINT
+// PRIMARY KEY (block, wtdrlIndex)
+
+func getWithdrawals(ctx context.Context, db *sql.DB, block uint64) ([]map[string]interface{}, error) {
+	query := fmt.Sprintf("SELECT withdrawls.wtdrlIndex, withdrawls.vldtrIndex, withdrawls.recipient, withdrawls.amount WHERE withdrawls.block = %v;", block)
```

Comment:
But also, it would be better to do `WHERE withdrawals.block = ?` and then add the block number as a parameter when you run `QueryContext`.

In general, formulating SQL statements with `Sprintf` needs to be done very carefully, and never with data provided to you by a user. https://xkcd.com/327/

Reply:
Lastly, "withdrawals" is misspelled multiple times in this line, while spelled correctly in the migrations file.

Reply:
Let's try to get a couple of simple tests going here that could at least exercise the SQL statements. I can help you craft some sample data, since we don't yet have a real chain to pull from.